lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,18 @@
+def create_commits!(repository)
+  repository.git.commits { |c| repository.commit_for!(c.oid) }
+end
+
+def attach_commit!(ontology_version)
+  commit = ontology_version.repository.commits.
+    find_by_commit_oid(ontology_version.commit_oid)
```

Comment:
And to be honest i actually find it easier to read. This way the reader knows, when reaching the end of the line, that the call-chaining is not over instead of being surprised later on.